test/tests/functional/rs_mocks/add_remove_tests.js

```diff
@@ -602,7 +602,7 @@ describe('ReplSet Add Remove (mocks)', function() {
             server.destroy();
             done();
           }, 1000);
-        }, 500);
+        }, 630); // This connection implementation slows down conneciton to nodes, need to fix this
```

Comment:
do you still need this change?

Reply:
Yes, this is still necessary, however, I am currently debugging the close event. I am hoping that closing at the proper time will speed up the connections